python/SeanZvvHbb13TeVSteps/README.md

```diff
@@ -0,0 +1,71 @@
+##Instructions for ZvvHbb on LXPLUS
+
+*Disclaimer: This package is under active development. The default branch, V20_from_silvio, is guaranteed to run at PSI or Pisa. The version that David uses runs on LXPLUS, but may or may not be synchronized with the default branch. The default branch is being tested on LXPLUS and, pending potential modifications and updates, should be verified to run on LXPLUS by the end of next week.*
```

Comment:
Hi @perrozzi, I'm not using lxbatch yet. I am still running the code "interactively" to track debug output. Once everything runs for me to first order, I'll work on ensuring they likewise work on lxbatch. From what I've read, there is a faster turnaround using the GRID than lxbatch. Is there a way to launch this as a crab job for future reference?

@silviodonato's convenience scripts call runAll.sh, which I understand and have modified before. As for submitThem.py, I have never called it, but I have read through it (in its more recent form where you have added the file list chunking).

As for diverging, I would not worry about that after this pull request. I had a limited time to tackle the problem of efficiently skimming and copying the files to lxplus, while keeping the workflow as general and safe as possible. That has been fully realized and debugged as of yesterday afternoon, allowing:
- @degrutto to use this code to prepare the ntuples for next week's exercise.
- The possibility of integrating this code into the current workflow in a fairly painless manner, following some clarifying discussions.
- The analysis workflow to remain undisturbed, as the output of the lxplus prep step is what the rest of the steps expect.